config/lib.json

```diff
@@ -421,11 +421,11 @@
     },
     "libjxl": {
         "source": "libjxl",
-        "static-libs-unix": [
-            "libjxl.a",
-            "libjxl_cms.a",
-            "libjxl_threads.a",
-            "libhwy.a"
+        "pkg-configs": [
+            "libjxl",
+            "libjxl_cms",
+            "libjxl_threads",
+            "libhwy"
```

Comment:
I think we only need to use pkgconfig of libjxl and the others should be in the output if needed.

Same goes for other libraries that don't have strictly decoupled components.

Reply:
Oh, I used pre-built gmp content, and it has `libgmp.la` with `-L/app/buildroot/lib`, and build with musl-toolchain will fail obviously.